scout/parse/variant/genotype.py

```diff
@@ -358,32 +359,26 @@ def get_alt_depth(
 ):
     """Get alternative read depth"""
     alt_depth = int(variant.gt_alt_depths[pos])
-    if alt_depth == -1:
-        if "VD" in variant.FORMAT:
-            alt_depth = int(variant.format("VD")[pos][0])
-
-        if any([paired_end_alt, split_read_alt]):
-            alt_depth = 0
-            if paired_end_alt:
-                alt_depth += paired_end_alt
-            if split_read_alt:
-                alt_depth += split_read_alt
-
-        if any([clip5_alt, clip3_alt]):
-            alt_depth = 0
-            if clip5_alt:
-                alt_depth += clip5_alt
-            if clip3_alt:
-                alt_depth += clip3_alt
-
-        if any([spanning_alt, flanking_alt, inrepeat_alt]):
-            alt_depth = 0
-            if spanning_alt:
-                alt_depth += spanning_alt
-            if flanking_alt:
-                alt_depth += flanking_alt
-            if inrepeat_alt:
-                alt_depth += inrepeat_alt
+    if alt_depth != -1:
```

Comment:
And his is != -1:

Reply:
```suggestion
    if alt_depth == -1:
```
Sry, missed that one!

Reply:
Return on that one, so this would be the correct one.. 🙄

Reply:
Sorry I son't understand. Shouldn't it return alt_depth if alt_depth != -1?

Reply:
Ahem yes? Oh.. Actually you do need copy paste! 😜